Project.toml

```diff
@@ -1,7 +1,7 @@
 name = "AxisKeys"
 uuid = "94b1ba4f-4ee9-5380-92f1-94cde586c3c5"
 license = "MIT"
-version = "0.1.5"
+version = "0.1.6"
```

Comment:
I don't think 0.1.5 was ever tagged?

Reply:
Oh I always mess that up, thanks, fixed now.